WalletWasabi.Documentation/FAQ.md

```diff
@@ -24,7 +24,7 @@ Note that for a coinjoin to happen there needs to be a sufficient number of part
 
 ### Will my coins be fully private after mixing with Wasabi?
 
-This depends on how you handle your outputs after the coinjoin. There are some ways how you can unintentionally undo the mixing by being careless. For example, if you make a 1.8 BTC transaction into Wasabi, do the coinjoin, and then make one single outgoing transaction of 1.8 BTC, a third party observer can reasonably assume that both transactions belong to one single entity, due to both amounts being virtually the same even though if they have been through a coinjoin. In this scenario, Wasabi will barely make any improvement to your privacy, it still have a protective effect against unsophisticated observers.  
+This depends on how you handle your outputs after the coinjoin. There are some ways how you can unintentionally undo the mixing by being careless. For example, if you make a 1.8 BTC transaction into Wasabi, do the coinjoin, and then make one single outgoing transaction of 1.8 BTC, a third party observer can reasonably assume that both transactions belong to one single entity, due to both amounts being virtually the same even though if they have been through a coinjoin. In this scenario, Wasabi will barely make any improvement to your privacy, although it still will have a protective effect against unsophisticated observers.  
```

Comment:
nit: there are extra spaces at the end of both lines

Reply:
Wow, good catch :)  
I think there are also extra spaces in other lines' endings. Is this something that should be corrected, can it cause troubles somewhere down the road with formatting etc?

Reply:
It doesn’t really matter, but it's better practice not to have the spaces

Reply:
Gotcha, will keep in mind, thanks. Must be a habit from markdown -.-